--- system-config-keyboard-1.3.1/lib/keyboard.py~	2012-03-03 18:27:49.000000000 +0200
+++ system-config-keyboard-1.3.1/lib/keyboard.py	2012-03-03 18:43:13.246317693 +0200
@@ -128,7 +128,12 @@
                 line = file.readline ()
                 if not line:
                         break
-                (name, value) = line.rstrip("\n").split ('=')
+                parts = line.rstrip("\n").split ('=')
+                if len(parts) >= 2:
+                    (name, value) = line.rstrip("\n").split ('=')
+                else:
+                    (name, value) = ("", "")
+
                 self.config.append ([line, name, value.strip ('"'), 0])
         self.beenset = 1
 
