--- system-config-keyboard-1.2.7/Makefile~	2004-09-07 11:44:34.000000000 +0300
+++ system-config-keyboard-1.2.7/Makefile	2006-09-28 02:58:07.924776759 +0300
@@ -30,13 +30,12 @@
 	mkdir -p $(INSTROOT)/usr/share/firstboot/modules
 	mkdir -p $(INSTROOT)/usr/share/icons/hicolor/48x48/apps
 	install src/*.py $(INSTROOT)$(PKGDATADIR)
-	install src/${PKGNAME} $(INSTROOT)$(PKGDATADIR)/${PKGNAME}
+	install src/${PKGNAME} $(INSTROOT)/usr/bin/${PKGNAME}
 	install pixmaps/*.png $(INSTROOT)$(PKGDATADIR)/pixmaps
 	install pixmaps/${PKGNAME}.png $(INSTROOT)/usr/share/icons/hicolor/48x48/apps
 	install ${PKGNAME}.pam $(INSTROOT)$(PAMD_DIR)/${PKGNAME}
 	install ${PKGNAME}.console $(INSTROOT)$(SECURITY_DIR)/${PKGNAME}
 	install ${PKGNAME}.desktop $(INSTROOT)/usr/share/applications/${PKGNAME}.desktop
-	ln -sf consolehelper $(INSTROOT)/usr/bin/${PKGNAME}
 	ln -sf $(PKGDATADIR)/keyboard_gui.py $(INSTROOT)/usr/share/firstboot/modules/keyboard.py 
 	for d in $(SUBDIRS); do \
 	(cd $$d; $(MAKE) INSTROOT=$(INSTROOT) MANDIR=$(MANDIR) install) \
