Index: src/keyboard_backend.py
===================================================================
--- src/keyboard_backend.py	(revision 562)
+++ src/keyboard_backend.py	(working copy)
@@ -36,9 +36,9 @@
             try:
                 keyboard = xf86config.getCoreKeyboard(xconfig)
             except:
-                xconfig.comment = '# This configuration file was broken by system-config-keyboard'
+                xconfig.comment = "\n# This configuration file was broken by system-config-keyboard\n"
                 keyboard = xf86config.XF86ConfInput ();
-                keyboard.comment = "# Keyboard added by system-config-keyboard"
+                keyboard.comment = "\n# Keyboard added by system-config-keyboard\n"
                 keyboard.identifier = "Keyboard0"
                 keyboard.driver = "kbd"
                 keyboard.options.insert (xf86config.XF86Option("XkbModel", "pc101"))
